lua/nvim-lsp-installer/servers/gopls.lua

```diff
@@ -0,0 +1,29 @@
+local server = require('nvim-lsp-installer.server')
+
+local root_dir = server.get_server_root_path('go')
+
+local install_cmd = [=[
+GO111MODULE=on GOBIN="$PWD" GOPATH="$PWD"  go get golang.org/x/tools/gopls@latest;
+
+if ! command -v ./gopls &> /dev/null; 
+then 
+  echo "something went wrong!"
+else 
+  echo "installation completed"
+fi
+
```

Comment:
I think this could be simplified to just:

```suggestion
command -v ./gopls &> /dev/null;
```

Reason being that the install scripts are executed with [`set -e`](https://github.com/williamboman/nvim-lsp-installer/blob/main/lua/nvim-lsp-installer/server.lua#L154), meaning it'll exit early if a command exits with a non-0 exit code. `command -v` will exit with code `1` if the specified command does not exist.

As for echoing feedback, this is already taken care of by the installer, [here](https://github.com/williamboman/nvim-lsp-installer/blob/main/lua/nvim-lsp-installer/server.lua#L159) and [here](https://github.com/williamboman/nvim-lsp-installer/blob/main/lua/nvim-lsp-installer/server.lua#L162)

Reply:
https://github.com/ecmma/nvim-lsp-installer/commit/4e461d619c24b52f3fae8797cd0333a1cfa6ac82 nice, thanks. 
Forgot to remove develop-time checks :)